tests/ui/deprecation/unit_and_tuple_struct.rs

```diff
@@ -0,0 +1,26 @@
+#![deny(deprecated)]
+#![allow(unused_imports)]
```

Comment:
Nit: this shouldn't be necessary, `unused` lints are disabled by default in failing ui tests.